onedal/dal.cpp

```diff
@@ -1,46 +1,39 @@
-/*******************************************************************************

```

Comment:
maybe rename dal.cpp -> modules.cpp for consistent with modules.hpp?

Reply:
I removed `modules.hpp` file and moved its content to `dal.cpp`